pkg/entities/default_role.go

```diff
@@ -25,6 +25,10 @@ func (e *Entity) GetDefaultRoleByGuildID(guildID string) (*response.DefaultRoleR
 }
 
 func (e *Entity) CreateDefaultRoleConfig(GuildID, RoleID string) error {
+	err := e.checkRoleIDBeenConfig(GuildID, RoleID)
```

Comment:
why need this here ?

Reply:
i think default role need to be different from other roles, so add here to check roleID has been used in any role configs, to make sure the default role is unique

Reply:
i think for now we dont need this flow, will add in future if needed